### ISSUE TITLE
Source

### ISSUE THREAD

**Issue by mlandmann** [Kademi/keditor#229]
Can you put the un minified js code back in the repository?
Also, where is this product going?  Do you expect to continue to enhance this over time?

**Comment by bradmac** [comment]
> Can you put the un minified js code back in the repository?

Thats here - https://github.com/Kademi/keditor/tree/master/src

>  Do you expect to continue to enhance this over time?

Yep! This is a critical capability in the sponsor company (https://kademi.co) so is being used and developed actively.

**Comment by mlandmann** [comment]
This is great news.

I think what has been put together here has some great potential.   

 

The one question if I might ask, is somewhere between version 1 and 2 the way it works changed.  There used to be a side panel you could put the blocks, etc. on and now it is the full screen drop down of select the various widgets.

Is this a configuration option, or is this the approach going forward?

 

I kinda preferred the old approach but have been warming up to the new one.

 

Thank you so much for responding!  

 

Mark Landmann

Mobile: 303-883-6196

Skype: mlandmann1

**Comment by bradmac** [comment]
>  Is this a configuration option, or is this the approach going forward?

The old approach was found to have major UX issues in practise, even though it looked nice and intuitive.

What we found is that if the content is small, and the number of components is small, then the old approach was great

But for more realistic use cases users need the full screen real estate to see and edit content, and the side bar was far too small to be useable. Eg in Kademi we have often around 200 components, and users cannot visually find the one they want in a small side bar.

What we realised is that the only time the side bar is relevant is when adding components. And when adding a component the user really wants to maximise screen real estate for finding the component they want

When users are not adding a component the side bar is irrelevant, so should be hidden to give maximum screen real estate for editing content.

I think there might be a config to show the sidebar, but not sure.
